index.js

```diff
@@ -188,7 +164,7 @@ function sendNotification(endpoint, params) {
         options.headers['Authorization'] = 'Bearer ' + jwt;
         var key = 'p256ecdsa=' + urlBase64.encode(vapid.publicKey);
         if (options.headers['Crypto-Key']) {
-          options.headers['Crypto-Key'] += ',' + key;
+          options.headers['Crypto-Key'] += ';' + key;
```

Comment:
/cc @martinthomson 

Which would, in turn, violate [draft-ietf-webpush-encryption](https://tools.ietf.org/html/draft-ietf-webpush-encryption-02#section-3.2):

> An application server MUST include exactly one entry in each of the Encryption and Crypto-Key header fields.

Semantically, having them in different entries makes sense because `p256ecdsa` has a different intended consumer (the push service) than `dh` (the user agent). This is unfortunately not compatible with all of today's implementations. FCM and Chrome enforce the single-entry restriction set by webpush-encryption, and will fail to deliver the message.

tl;dr— I suggest using `;` for compatibility reasons until we [fix the spec](https://github.com/webpush-wg/webpush-encryption/issues/6) and know the implementations can deal with this.